<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1025</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>krbkACD (filemanager)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="FileManagerPage" name="fsManagerPage"/>
      <widget class="ImageViewManager" name="ivManagerPage"/>
      <widget class="MetadataTreeViewPage" name="metadataViewPage"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1025</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>ACD</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionShowIcons"/>
    <addaction name="actionShowHidden"/>
    <addaction name="actionShowSecondBrowser"/>
    <addaction name="actionShowInfo"/>
    <addaction name="separator"/>
    <addaction name="actionNextFile"/>
    <addaction name="actionPreviousFile"/>
    <addaction name="separator"/>
    <addaction name="actionRenameFiles"/>
    <addaction name="actionCopyFiles"/>
    <addaction name="actionMoveFiles"/>
    <addaction name="actionDeleteFiles"/>
    <addaction name="actionMkDir"/>
   </widget>
   <widget class="QMenu" name="menuView_2">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionViewImage"/>
    <addaction name="actionFullScreen"/>
   </widget>
   <widget class="QMenu" name="menuMetadata">
    <property name="title">
     <string>Metadata</string>
    </property>
    <addaction name="actionViewMetadata"/>
    <addaction name="actionEditXmpLocation"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuView_2"/>
   <addaction name="menuMetadata"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionShowIcons">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Icons</string>
   </property>
  </action>
  <action name="actionShowHidden">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Hidden</string>
   </property>
  </action>
  <action name="actionShowSecondBrowser">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show 2nd file browser</string>
   </property>
  </action>
  <action name="actionCopyFiles">
   <property name="text">
    <string>Copy File(s)</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionMoveFiles">
   <property name="text">
    <string>Move File(s)</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionDeleteFiles">
   <property name="text">
    <string>Delete File(s)</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionRenameFiles">
   <property name="text">
    <string>Rename File(s)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionMkDir">
   <property name="text">
    <string>Create Directory</string>
   </property>
   <property name="shortcut">
    <string>F7</string>
   </property>
  </action>
  <action name="actionNextFile">
   <property name="text">
    <string>Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPreviousFile">
   <property name="text">
    <string>Previous</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionShowInfo">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show Info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionViewImage">
   <property name="text">
    <string>View Image</string>
   </property>
   <property name="toolTip">
    <string>View Image</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionViewMetadata">
   <property name="text">
    <string>View Metadata</string>
   </property>
   <property name="toolTip">
    <string>View Metadata</string>
   </property>
   <property name="shortcut">
    <string>F2</string>
   </property>
  </action>
  <action name="actionFullScreen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Full Screen</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionViewImageRegions">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Image Regions</string>
   </property>
  </action>
  <action name="actionEditXmpLocation">
   <property name="text">
    <string>Edit Xmp Location</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileManagerPage</class>
   <extends>QWidget</extends>
   <header location="global">FileManagerPage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImageViewManager</class>
   <extends>QWidget</extends>
   <header location="global">ImageViewManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MetadataTreeViewPage</class>
   <extends>QWidget</extends>
   <header>MetadataTreeViewPage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
